src/signals/incident-management/__tests__/selectors.test.js

```diff
@@ -100,6 +104,66 @@ describe('signals/incident-management/selectors', () => {
     expect(makeSelectEditFilter(state).id).toEqual(filters[2].id);
   });
 
+  it('should select page', () => {
+    const emptState = fromJS({
+      incidentManagement: { ...initialState.toJS() },
+    });
+    expect(makeSelectPage(emptState)).toEqual(initialState.toJS().page);
+
+    const state = fromJS({
+      incidentManagement: { ...initialState.toJS(), page: 100 },
+    });
+
+    expect(makeSelectPage(state)).toEqual(100);
+  });
+
+  it('should select ordering', () => {
+    const emptState = fromJS({
+      incidentManagement: { ...initialState.toJS() },
+    });
+    expect(makeSelectOrdering(emptState)).toEqual(initialState.toJS().ordering);
+
+    const state = fromJS({
+      incidentManagement: { ...initialState.toJS(), ordering: 'some-ordering-type' },
+    });
+
+    expect(makeSelectOrdering(state)).toEqual('some-ordering-type');
+  });
+
+  it('should select incidents', () => {
+    const emptState = fromJS({
+      incidentManagement: { ...initialState.toJS() },
+    });
+    expect(makeSelectIncidents(emptState)).toEqual(initialState.toJS().incidents);
+
+    const results = [...new Array(10).keys()].map(index => ({
+      id: index + 1,
+    }));
+
+    const incidents = { count: 100, results }
+
+    const state = fromJS({
+      incidentManagement: { ...initialState.toJS(), loading: false, incidents },
+    });
+
+    expect(makeSelectIncidents(state)).toEqual({ ...incidents, loading: false });
```

Comment:
Seems to be missing a check for when loading is `undefined`

Reply:
Good catch; `loading` isn't even in the initial state. I'll add it and change the tests accordingly.